en/lessons/advanced/gen-stage.md

```diff
@@ -247,10 +247,16 @@ Let's make some adjustments for multiple workers by modifying `lib/genstage_exam
 
 ```elixir
 children = [
-  worker(GenstageExample.Producer, [0]),
-  worker(GenstageExample.ProducerConsumer, []),
-  worker(GenstageExample.Consumer, [], id: 1),
-  worker(GenstageExample.Consumer, [], id: 2)
+  {GenstageExample.Producer, 0},
+  {GenstageExample.ProducerConsumer, []},
+  %{
```

Comment:
I don't think there is another way of passing the id without using a full on child spec struct like this.